samples/simple_txrx/src/rail-configs/sltb010a/radio_settings.radioconf

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<multi_phy_configuration part_family="lynx" part_revision="A0" rail_adapter_version="rail_api_2.x" status_code="0" xsd_version="0.0.20">
+  <base_channel_configurations>
+    <base_channel_configuration name="Protocol Configuration" profile="Base">
+      <channel_config_entries>
+        <channel_config_entry name="Channel Group 1">
+          <channel_number_start>0</channel_number_start>
+          <channel_number_end>20</channel_number_end>
+          <physical_channel_offset>SAME_AS_FIRST_CHANNEL</physical_channel_offset>
+          <max_power>RAIL_TX_POWER_MAX</max_power>
+          <metadata>{"selectedPhy":"PHY_Datasheet_2450M_2GFSK_1Mbps_500K"}</metadata>
+        </channel_config_entry>
+      </channel_config_entries>
+      <metadata>{"selectedPhy":"PHY_Datasheet_2450M_2GFSK_1Mbps_500K"}</metadata>
+      <phy name="PHY_Datasheet_2450M_2GFSK_1Mbps_500K"/>
+    </base_channel_configuration>
+  </base_channel_configurations>
+</multi_phy_configuration>
```

Comment:
GitHub seems to complain of a missing newline character at the end of the file

Reply:
I believe GitHub does not complain, it just notifies.

Since this file is generated by a tool, I hesitate to edit it. That's I have already applied (too) many changes to `rail-config.[ch]`.